ui/apps/dashboard/src/components/Metrics/Volume.tsx

```diff
@@ -217,12 +216,12 @@ export const MetricsVolume = ({
             <RunsThrougput workspace={data?.workspace} entities={entities} />
             <StepsThroughput workspace={data?.workspace} entities={entities} />
             <div className="col-span-2 flex flex-row flex-wrap gap-2 overflow-hidden md:flex-nowrap">
-              <SdkThroughput workspace={data?.workspace} entities={entities} />
+              {/* <SdkThroughput workspace={data?.workspace} entities={entities} /> */}
               <Backlog workspace={data?.workspace} entities={entities} />
+              <Feedback />
             </div>
             <div className="col-span-2 flex flex-row flex-wrap gap-2 overflow-hidden md:flex-nowrap">
-              <AccountConcurrency workspace={data?.workspace} entities={entities} />
-              <Feedback />
+              {/* <AccountConcurrency workspace={data?.workspace} entities={entities} /> */}
```

Comment:
Is this commented and not deleted for a particular reason?

Reply:
We just pulled it from the launch, I can delete it.

Reply:
ok, they are deleted.